src/xcode/ENA/ENAUITests/HealthCertificate/ENAUITests_13_CreateHealthCertificate.swift

```diff
@@ -575,7 +575,9 @@ class ENAUITests_13_CreateHealthCertificate: CWATestCase {
 		app.launch()
 
 		// Navigate to Certificates Tab.
-		app.buttons[AccessibilityIdentifiers.TabBar.certificates].waitAndTap()
+		let certificateTab = app.buttons[AccessibilityIdentifiers.TabBar.certificates]
+		XCTAssertTrue(certificateTab.waitForExistence(timeout: .medium))
+		certificateTab.waitAndTap()
```

Comment:
waitAndTap() is already doing what's in line 579. You can just pass .medium as timeout.

Reply:
i updated the code and it works locally, thanks